prow/config.yaml

```diff
@@ -367,12 +367,15 @@ presubmits:
     trigger: "(?m)^/test pull-kubernetes-e2e-gce-gpu-prow,?(\\s+|$)"
     spec:
       containers:
-      - args:
+      - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170829-93ec4e74
```

Comment:
(current version is kubekins-e2e-prow:v20170913-207d52da, and https://github.com/kubernetes/test-infra/pull/4620 will bump it to kubekins-e2e:v20170918-3aa6c113)

Reply:
bumped to match #4620